### PR TITLE
LPD-26938

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-26938)

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_gl.properties [hunk 1]
```diff
@@ -3107,7 +3107,7 @@ category.batch-planner=Planificador por lotes
 category.bi=Intelixencia empresarial e informes
 category.blogs=Blogs
 category.breadcrumbs=Camiños de navegación
-category.captcha=Captcha
+category.captcha=CAPTCHA
```
Comment (igor-franca) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_gl.properties:3110]: This change is related to some modification in `language.properties` that did not execute the git add of the `Language_gl.properties` file after BuildLang

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by igor-franca** [comment]
ci:stop

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**